src/arithmetic/path_funcs/path_funcs.c

```diff
@@ -20,15 +20,6 @@
  */
 static void _PathBuilder_AppendNode(Path *path, Node *node) {
```

Comment:
I don't mind changing it but, I like it better this way. Reading lines like
```
for(uint i = 0; i < new_path_edge_count - 1; i++) {
		// Insert only nodes which are not the last and the first, since they will be added by append node specifically.
		_PathBuilder_AppendNode(path, &new_path->nodes[i + 1]);
		_PathBuilder_AppendEdge(path, &new_path->edges[i], RTLEdge);
	}
```
is more comprehensive, in my opinion of course. 
change?

Reply:
I think `Path_AppendNode` is clear enough
```for(uint i = 0; i < new_path_edge_count - 1; i++) {
		// Insert only nodes which are not the last and the first, since they will be added by append node specifically.
		Path_AppendNode(path, &new_path->nodes[i + 1]);
		Path_AppendNode(path, &new_path->edges[i], RTLEdge);
	}```